src/lib/support/logging/Constants.h

```diff
@@ -5,6 +5,53 @@
 namespace chip {
 namespace Logging {
 
+// X-Macro style enumeration of log modules and associated short names
+// Note: For any module added here, a CHIP_CONFIG_LOG_MODULE_* macro needs to be defined below.
```

Comment:
Add usage example here and also explain design choices:
- we want to explicitly provide "Module long name" and "module 3-letter name" and let others build
- this allows platforms to choose to show short or long name.